tests/test_cv.py

```diff
@@ -43,6 +63,20 @@ def tearDown():
     if exists(fold_file_path):
         os.unlink(fold_file_path)
 
+    train_dir = join(_my_dir, 'train')
+    output_dir = join(_my_dir, 'output')
+    config_dir = join(_my_dir, 'configs')
+
+    cfg_file = join(config_dir, 'test_save_cv_folds.cfg')
+    os.unlink(cfg_file)
```

Comment:
Just wondering: Is there some specific reason we're using unlink here as opposed to remove or is it just because that's what has been used in the past? The docs say that it's identical to remove and it seems like the naming is clearer there.